Userland/Libraries/LibSQL/Type.h

```diff
@@ -12,17 +12,17 @@
 
 namespace SQL {
 
-#define ENUMERATE_SQL_TYPES(S)                             \
-    S("null", 1, Null, int, sizeof(int))                   \
-    S("text", 2, Text, DeprecatedString, 65 + sizeof(u32)) \
-    S("int", 4, Integer, int, sizeof(int))                 \
-    S("float", 8, Float, double, sizeof(double))           \
-    S("bool", 16, Boolean, bool, sizeof(bool))             \
-    S("tuple", 32, Tuple, int, sizeof(int))
+#define ENUMERATE_SQL_TYPES(S) \
+    S("null", Null)            \
+    S("text", Text)            \
+    S("int", Integer)          \
+    S("float", Float)          \
+    S("bool", Boolean)         \
+    S("tuple", Tuple)
 
 enum class SQLType {
 #undef __ENUMERATE_SQL_TYPE
-#define __ENUMERATE_SQL_TYPE(name, cardinal, type, impl, size) type = cardinal,
```

Comment:
Is this change compatible with the rest of LibSQL on its own?
Since the values of the enum changed? Or should this be a version change?
(Mind you I don't know anything about LibSQL)

Reply:
Hmm yeah it's technically a version change. The values themselves don't matter, but once a type is written on anyone's disk, changing its value means we won't be able to read that DB anymore. So I'll do a version bump in this commit as well, and add a comment warning about it.